docs/contributing/docs-contributions.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Docs Contributions
+---
+
+Gatsby, unsurprisingly, uses Gatsby for its documentation website. Thank you in advance and cheers for contributing to Gatsby documentation! As of February 2019, over 800 people have contributed. It's people like you that make this community great!
```

Comment:
Was this number targeted somehow to just docs contributions? Otherwise, it's about double!

```suggestion
Gatsby, unsurprisingly, uses Gatsby for its documentation website. Thank you in advance and cheers for contributing to Gatsby documentation! As of February 2019, over 1600 people have contributed. It's people like you that make this community great!
```